### DISCUSSION TITLE
ORJSONResponse options

### DISCUSSION THREAD

**Discussion by plankter** [fastapi/fastapi#7687] [Questions] [answered]
Please add a functionality to pass `orjson` options (https://github.com/ijl/orjson#option) to ORJSONResponse. For instance, `OPT_SERIALIZE_NUMPY` is quite useful in many scenarios.

**Comment by raphaelauv** [comment]
it's possible and easily by doing this :

#1846 (comment)

You can probably close this issue, thank you

**Comment by raphaelauv** [comment]
@plankter 

Can you close this issue since it's easily possible by extending the current class. Thank you

**Comment by tiangolo** [comment, marked as answer]
Thanks for the interest @plankter! And thanks a lot for the help here @raphaelauv! :bow: 

I added this section to the docs explaining how to do that: https://fastapi.tiangolo.com/advanced/custom-response/#custom-response-class
